kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/AppConfig.java

```diff
@@ -0,0 +1,81 @@
+package com.kodilla.good.patterns.food2door;
+
+import com.kodilla.good.patterns.food2door.data.Company;
+import com.kodilla.good.patterns.food2door.demands.*;
+import com.kodilla.good.patterns.food2door.process.*;
+import com.kodilla.good.patterns.food2door.services.*;
+
+import java.util.*;
+
+// dummy implementation of config class. Normally it should read data from config file, for example config.xml
+public final class AppConfig {
+    private Company appUser;
+    private DemandRetriever demandRetriever;
+    private  StorageService storageService;
+    private  Map<Company, OfferService> offerServices;
+    private  Map<Company, OrderService> orderServices;
+    private OrderProcessorResultHandler orderProcessorResultHandler;
+
+    public AppConfig(String configFile) {
```

Comment:
If it is dummy config, do not pretend that data is read from configFile - it is misleading.